### ISSUE TITLE
iOS version prerequisite

### ISSUE THREAD

**Issue by gilles-leblanc** [MicrosoftDocs/appcenter-docs#83]
Section 1 Prerequisites mentions:
"You are targeting devices running iOS 9.0 or later or Android 4.0.3 (API level 15) or later".
In my info.plist my Deployment Target is 9.0.

Yet when I add the 1.5.0 AppCenter packages I am getting the following errors:
"error MT5209 : Native linking error : warning: object file  was built for newer iOS version (10.1) than being linked (9.0)".

Is the 9.0 or later prerequisite still accurate?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5d72f6fe-74d6-696e-e709-5f6d577c0a58
* Version Independent ID: b3d0ea8b-5a1f-bf71-7dc2-9685cf23c00b
* Content: [Get Started with Xamarin - Visual Studio App Center](https://docs.microsoft.com/en-us/appcenter/sdk/getting-started/xamarin)
* Content Source: [docs/sdk/getting-started/xamarin.md](https://github.com/MicrosoftDocs/appcenter-docs/blob/live/docs/sdk/getting-started/xamarin.md)
* Service: **vs-appcenter**
* GitHub Login: @elamalani
* Microsoft Alias: **emalani**

**Comment by elamalani** [comment]
@gilles-leblanc App Center supports iOS 9 and later. If you see an error adding App Center NuGet, can you open a support ticket with us by clicking on the blue icon in the lower right of App Center portal? That way,  we can better help you investigate the issue.

**Comment by gilles-leblanc** [comment]
I've found out that if I only add the AppCenter base package this does not happen. 
I need to add the AppCenter.Crashes as well to get the errors.

I then tried adding only the AppCenter and AppCenter.Analytics package and it worked. 
If I add the AppCenter.Crashes package again it starts failing again.

So I tested out all these combinations:

- AppCenter = ok
- AppCenter + AppCenter.Crashes = errors
- AppCenter + AppCenter.Analytics = ok
- AppCenter + AppCenter.Crashes + AppCenter.Analytics = errors

After looking more carefully at the errors I noticed something about Hockey in the logs (which I am also using). 

So I removed the Hockey package for my iOS project and everything works.
My bad.

I also later saw that it mentioned not to use both at the same time in the documentation.
